src/constants/actions.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // action types
+  TRANSITION: 'ACTION_TYPE/TRANSITION',
+  METADATA: 'ACTION_TYPE/METADATA',
+  PROGRESS: 'ACTION_TYPE/PROGRESS',
+  // transition actions
+  REJECT: 'TRANSITION/REJECT',
```

Comment:
Then what would the routings be like?
Ex. `pin/{id}/state_transition/reject`  to set state reject?